files/setup-01-os.sh

```diff
@@ -21,3 +21,7 @@ if [ "${DOCKER_NETWORK_CIDR}" != "172.17.0.1/16" ]; then
 EOF
 systemctl restart docker
 fi
+
+echo -e "\e[92mConfiguring IP Tables for Antrea ..." > /dev/console
+iptables -A INPUT -i gw0 -j ACCEPT
```

Comment:
What is `gw0` interface? Something Antrea uses internally or exposed to the outside?

Reply:
Yup, its an Antrea internal interface and FW rule is required

Per [Antrea Architecture](https://github.com/vmware-tanzu/antrea/blob/master/docs/architecture.md)
> On the OVS bridge, Antrea Agent also creates an internal port - gw0 by default - to be the gateway of the Node's subnet, and a tunnel port tun0 which is for creating overlay tunnels to other Nodes.